fdbserver/CMakeLists.txt

```diff
@@ -194,7 +194,9 @@ set(FDBSERVER_SRCS
   workloads/WorkerErrors.actor.cpp
   workloads/workloads.actor.h
   workloads/WriteBandwidth.actor.cpp
-  workloads/WriteDuringRead.actor.cpp)
+  workloads/WriteDuringRead.actor.cpp
+  ${PROJECT_SOURCE_DIR}/bindings/c/test/mako/zipf.h
+  ${PROJECT_SOURCE_DIR}/bindings/c/test/mako/zipf.c)
```

Comment:
Could you please move these files to flow/, and then via depending on flow from both bindings and fdbserver should mean that it's linked into both artifacts?

Reply:
fdbclient/ may be a more natural place for these, and it's already accessible to fdbserver.

Reply:
Will do. Thanks.

Reply:
Done. Thanks.